src/12_scopes.py

```diff
@@ -6,6 +6,7 @@
 
 def change_x():
     x = 99
+    print(x)
```

Comment:
Rather than moving the print statement, please read through [this article](https://www.programiz.com/python-programming/global-local-nonlocal-variables)